horovod/spark/common/params.py

```diff
@@ -156,6 +156,8 @@ def getStore(self):
         return self.getOrDefault(self.store)
 
     def setLoss(self, value):
+        if not isinstance(value, list):
```

Comment:
I'm not a fan of this solution because it results in counter-intuitive behavior:

```
estimator.setLoss(loss)
estimator.getLoss() == loss  # False, but expected True
```

We should do the wrapping etc. only at train time.

Reply:
I see your point, will revert back.